HerPortal.BusinessLogic/Models/ConsortiumData.cs

```diff
@@ -1,7 +1,15 @@
 ﻿namespace HerPortal.BusinessLogic.Models;
 
+/// <summary>
+///     LA-Consortium mapping data can be found both in <see cref="LocalAuthorityData"/>, and in the HUG2 Public Website codebase's LocalAuthorityData.cs.
+///     <seealso href="https://github.com/UKGovernmentBEIS/desnz-home-energy-retrofit-beta/blob/develop/HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs">Link to HUG2 Public Website codebase's LocalAuthorityData.cs</seealso>
+///     Note: Mappings in the above files are expected to be consistent between both repos
+/// </summary>
 public static class ConsortiumData
 {
+    /// Ensure that the code mapping in <see cref="LocalAuthorityData"/> is updated if any custodian codes change
+    /// If a Consortium Code changes, we will need to perform a migration to map old users of the code to
+    /// use the new code, and begin using the new code.
```

Comment:
Realise this is off Sammy's point, but I was being deliberately vague with this because I think there would be a few options available to us. For example, see `LaMapping` in the main repo: could do something like that instead. Think it's enough to say "if changes we need to think about it" in so many words.

For consortium codes specifically, as far as I can tell they're an entirely internal thing anyway, so we shouldn't need to worry about them changes.